File: src/java/org/javaturk/dp/ch02/elevator/domain/GetterSetterElevator.java
package org.javaturk.dp.ch02.elevator.domain;

public class GetterSetterElevator {

	private boolean doorOpen = false;
	private int currentFloor = 1;
	private int weight = 0;

	private final int CAPACITY = 1000;
	private final int TOP_FLOOR = 5;
	private final int BOTTOM_FLOOR = 1;

	public boolean isDoorOpen() {
		return doorOpen;
	}

	public void setDoorOpen(boolean doorOpen) {
		this.doorOpen = doorOpen;
	}

	public int getCurrentFloor() {
		return currentFloor;
	}

	public void setCurrentFloor(int currentFloor) {
		this.currentFloor = currentFloor;
	}

	public int getWeight() {
		return weight;
	}

	public void setWeight(int weight) {
		this.weight = weight;
	}

	public int getCAPACITY() {
		return CAPACITY;
	}

	public int getTOP_FLOOR() {
		return TOP_FLOOR;
	}

	public int getBOTTOM_FLOOR() {
		return BOTTOM_FLOOR;
	}
}

File: src/c#/ch02/Elevator/Domain/GetterSetterElevator.cs
using System;
namespace Elevator.Domain {
    public class GetterSetterElevator {
		public bool DoorOpen { get; set; }  = false;
		public int CurrentFloor { get; set; } = 1;
		public int Weight { get; set; } = 0;

		public readonly int CAPACITY = 1000;
		public readonly int TOP_FLOOR = 5;
		public readonly int BOTTOM_FLOOR = 1;
	}
}
